scripts/pipeline_generator/step.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import BaseModel, Field
+from typing import List, Dict, Any, Optional
+
+from .utils import AgentQueue
+
+
+class BuildkiteBlockStep(BaseModel):
+    """This class represents a block step in Buildkite format."""
+    block: str
+    depends_on: Optional[str] = "build"
```

Comment:
maybe declare a global constant for `"build"`?

Reply:
done